tests/at_end2end_test/test/notify_verb_test.dart

```diff
@@ -592,16 +609,34 @@ void main() {
     print('notify status response : $response');
     expect(response, contains('data:delivered'));
 
-    await sh2.writeCommand('llookup:all:cached:$atSign_2:phone.wavi$atSign_1');
+    await sh2.writeCommand('llookup:all:cached:$atKey');
     response = await sh2.read();
     response = response.replaceAll('data:', '');
     var decodedResponse = jsonDecode(response);
-    expect(decodedResponse['key'], 'cached:$atSign_2:phone.wavi$atSign_1');
+    expect(decodedResponse['key'], 'cached:$atKey');
     expect(decodedResponse['data'], 'Some ciphertext');
     expect(decodedResponse['metaData']['sharedKeyEnc'], 'abc');
     expect(decodedResponse['metaData']['pubKeyCS'],
         '3c55db695d94b304827367a4f5cab8ae');
-    expect(decodedResponse['metaData']['ttr'], 60000);
+    expect(decodedResponse['metaData']['ttr'], 60);
+    expect(decodedResponse['metaData']['ttl'], 60000);
+    expect(PublicKeyHash.fromJson(decodedResponse['metaData']['pubKeyHash']),
+        somePubKeyHash);
+    expect(decodedResponse['metaData']['immutable'], expectedImmutable);
+
+    // Let's try to delete the cached record on the recipient side.
+    // Cached data should *always* be deletable regardless of immutable flag.
```

Comment:
extend the e2e test so that we verify that the recipient can delete the cached record even if the originator supplied 'immutable:true' in the notification metadata